lib/open-amp/resource_table.h

```diff
@@ -16,17 +16,23 @@ extern "C" {
 
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
-#define VDEV_ID                 0xFF
-#define VRING0_ID 0 /* (master to remote) fixed to 0 for Linux compatibility */
-#define VRING1_ID 1 /* (remote to master) fixed to 1 for Linux compatibility */
-
-#define VRING_COUNT             2
-#define RPMSG_IPU_C0_FEATURES   1
-
-#define VRING_RX_ADDRESS        -1  /* allocated by Master processor */
-#define VRING_TX_ADDRESS        -1  /* allocated by Master processor */
-#define VRING_BUFF_ADDRESS      -1  /* allocated by Master processor */
-#define VRING_ALIGNMENT         16  /* fixed to match with Linux constraint */
+#define VDEV_ID 0xFF
+#define VRING0_ID                                                                                  \
+	CONFIG_OPENAMP_RSC_TABLE_IPM_RX_ID /* (master to remote) default to 0 for                  \
+					    * Linux compatibility                                  \
+					    */
+#define VRING1_ID                                                                                  \
+	CONFIG_OPENAMP_RSC_TABLE_IPM_TX_ID /* (remote to master) default to 1 for                  \
+					    * Linux compatibility                                  \
+					    */
+
+#define VRING_COUNT           2
+#define RPMSG_IPU_C0_FEATURES 1
+
+#define VRING_RX_ADDRESS   -1 /* allocated by Master processor */
+#define VRING_TX_ADDRESS   -1 /* allocated by Master processor */
+#define VRING_BUFF_ADDRESS -1 /* allocated by Master processor */
+#define VRING_ALIGNMENT    16 /* fixed to match with Linux constraint */
 
```

Comment:
I think you changed something here so that it appears in the diff.

```
-#define VRING_ALIGNMENT         16  /* fixed to match with Linux constraint */
+#define VRING_ALIGNMENT    16 /* fixed to match with Linux constraint */
```

This is not what we want. If you want to change the spacing or anything else we can later create a separate patch.